pkgs/by-name/en/envision-unwrapped/package.nix

```diff
@@ -81,7 +96,23 @@ stdenv.mkDerivation (finalAttrs: {
       --prefix PATH : "${lib.makeBinPath [ gdb ]}"
   '';
 
-  passthru.updateScript = nix-update-script { };
+  passthru = {
+    updateScript = nix-update-script { };
+    tests = {
+      version = testers.testVersion {
```

Comment:
Can you test with this?

https://nixos.org/manual/nixpkgs/stable/#versioncheckhook

That way this test will always run on package build time